src/compiler/Compiler.js

```diff
@@ -92,7 +92,13 @@ module.exports = class Compiler extends EventEmitter {
    * Create fork process and attach necessary event listeners.
    */
   initFork({ platform, options }: { platform: Platform, options: * }) {
-    const fork = new Fork({ platform, options });
+    let fork;
+    try {
+      fork = new Fork({ platform, options });
+    } catch (err) {
+      this.emit(Events.BUILD_FAILED, { platform, message: err.toString() });
```

Comment:
`toString` is probably unnecessary and should be done automatically when sending to worker.

Reply:
I would rather be explicit.

Reply:
It's pointless. the whole thing will be stringified in the end.

Reply:
`toString()` will resolve to `message` if the throwned object is an instance of Error. `message` could break the code if someone throw something different. But fair point about stringified I'm going to change it.